privacyidea/lib/config.py

```diff
@@ -43,6 +43,7 @@
 from .log import log_with
 from ..models import (Config, db, Resolver, Realm, PRIVACYIDEA_TIMESTAMP,
                       save_config_timestamp, Policy, EventHandler)
+from privacyidea.lib.error import ValidateError
```

Comment:
I think we should use a `ParameterError`. The `ValidateError` is used, if the some error occurred during the authentication process.

In a normal authentication process you the application would not get the idea to ask for a non-existing tokentype, so this is rather an application passing wrong parameters.

Reply:
Done, good input. I also thought about ParameterError but didn't find an explanation.
Changed.